core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraService.java

```diff
@@ -19,41 +19,23 @@
 import io.trino.server.ui.WebUiModule;
```

Comment:
I'm guessing this just makes it simpler, since we only need one container? We aren't testing Hydra's integration with PostgreSQL, so it's fine as long as Hydra works the same for tests.

Reply:
The latest Hydra release (https://github.com/ory/hydra/releases/tag/v1.9.0) introduced:
`Support for SQLite, an embedded database, which can be used for testing and tiny deployments.`
and it's the feature I'm trying to use here.